fs/dafilesrv/dafilesrv.cpp

```diff
@@ -44,17 +45,19 @@ static const char* defaultRowSericeConfiguration = "RowSvc";
 void usage()
 {
     printf("dafilesrv usage:\n");
-    printf("    dafilesrv [-T<n>] [...] [<port>] [<send-buff-size-kb> <recv-buff-size-kb>]\n");
+    printf("    dafilesrv [--trace <n>] [...] [--port <port>] [--sbSize <send-buff-size-kb>] [--rbSize <recv-buff-size-kb>]\n");
```

Comment:
I like the new argument names, but is there any backward compatibility issue ?

Reply:
the init script has also been changed to reflect the new arguement list style,
so I do not think there are any compatibility issues.
It's possible Windows users (if they ever used a new version of dafilesrv) would have to amend their own manual scripts.